src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-wrapper.service.ts

```diff
@@ -51,7 +51,7 @@ export class TaskWrapperService {
     this.notificationService.show(
       NotificationType.info,
       this.taskManagerMessageService.getRunningMessage(task),
-      this.taskManagerMessageService.getDescription(task)
+      this.taskManagerMessageService.getErrorTitle(task)
```

Comment:
This can be remove.
We no longer need a message for this notification and atm it is showing an error message.

Reply:
We still need the title for consistency among all other tasks. This way every notification has a title, no mixing like before.

Reply:
The title is the line above the one I'm suggesting we remove.
The ErrorTitle is being passed as the message.